### ISSUE TITLE
Add filters and queries for admin users

### ISSUE THREAD

**Issue by jlin816** [techx/quill#7]
By:
- [ ] verified
- [ ] submitted
- [ ] accepted
- [ ] confirmed
- [ ] needsReimbursement

**Comment by cmnord** [comment]
Would this allow you to email everyone checked-in, etc?

**Comment by cmnord** [comment]
Or get the emails of everyone accepted but not confirmed?

**Comment by jlin816** [comment]
Nope, I was just envisioning that you'd be able to filter your searches. Exporting emails/other info of any particular search would be a separate feature that would be nice to have -- put in another issue! :)
